### ISSUE TITLE
dndz initilization in Tracer object

### ISSUE THREAD

**Issue by marcobonici** [cloe-org/cloelib#24]
In the `Tracer` is written

```python
super().__init__(perturbations)
self.dndz = np.vstack(list(dndz.values()))
```
However, I don't like this choice, which I think requires a discussion. This method will work, but is tied to `euclidlib`. While this is not wrong per se, I think we should not make such a choice, because a user might come with a more general structure. I think the right kind of input should be a `numpy` matrix, with one axis representing the tomographic bin and the other one the redshift (which, btw, is also what we are assigning to the `dndz` field of the `Tracer` object).

**Comment by gcanasherrera** [comment]
I agree. Maybe it would be worth changing the format in `euclidlib`.

**Comment by gcanasherrera** [comment]
Adding here the link to the discussion for the feature in `euclidlib`: https://github.com/euclidlib/euclidlib/issues/9

**Comment by gcanasherrera** [comment]
This is no longer relevant
